pkg/harvester/l10n/en-us.yaml

```diff
@@ -352,6 +352,7 @@ harvester:
     enableMaintenance:
       title: Enable Maintenance Mode
       protip: The operation will migrate all virtual machines on this node to other nodes.
+      shoutDownVMs: You can click <b>Force</b> to automatically shutdown non-migratable VMs and continue with enabling Maintenance Mode
```

Comment:
Do you think using `check` is more specific ?
```
You can check <b>Force</b> to automatically shutdown non-migratable VMs and continue with enabling Maintenance Mode 
```

Reply:
I will reword the message